pybamm/models/submodels/thermal/x_lumped/x_lumped_1D_current_collectors.py

```diff
@@ -15,11 +15,16 @@ def __init__(self, param):
     def set_rhs(self, variables):
         T_av = variables["X-averaged cell temperature"]
         Q_av = variables["X-averaged total heating"]
+        T_amb = variables["Ambient temperature"]
 
         cooling_coeff = self._surface_cooling_coefficient()
 
         self.rhs = {
-            T_av: (pybamm.laplacian(T_av) + self.param.B * Q_av + cooling_coeff * T_av)
+            T_av: (
+                pybamm.laplacian(T_av - T_amb)
+                + self.param.B * Q_av
+                + cooling_coeff * (T_av - T_amb)
+            )
             / self.param.C_th
         }
```

Comment:
just thought - for the models that are x-lumped with 1D and 2D current collectors I think you also need to change the boundary conditions to `T-T_amb` as well